templates/components/carousel.html

```diff
@@ -10,10 +10,12 @@
     }'>
     {{#each carousel.slides}}
     <a href="{{url}}">
-        <div class="heroCarousel-slide {{#if ../theme_settings.homepage_stretch_carousel_images}}heroCarousel-slide--stretch{{/if}}"
+        <div class="heroCarousel-slide {{#if ../theme_settings.homepage_stretch_carousel_images}}heroCarousel-slide--stretch{{/if}} {{#if @first}}heroCarousel-slide--first{{/if}}"
             style="background-image: url({{image}})">
-            <img class="heroCarousel-image" data-lazy="{{image}}" alt="{{alt_text}}" title="{{alt_text}}""
-             {{#if image_width}}width="{{image_width}}"{{/if}} {{#if image_height}}height="{{image_height}}"{{/if}} />
+            <div class="heroCarousel-image-wrapper" style="height:{{multiply (divide image_height image_width) 100}}vw">
```

Comment:
What if image_width or image_height are not present? I'm assuming that is possible since below we have conditionals for them.

Reply:
Do we need this extra div, or can we use the slide div?

Reply:
We can't solely use the slide div, as there is more content below the image in the case of mobile (rather than overlaying the text and buttons over the banner, mobile displays them below the banner)

Reply:
(But yeah, good point re: the image width and height, though I'm not sure when that happens.)